import "goby/common/protobuf/option_extensions.proto";
import "goby/common/protobuf/app_base_config.proto";
import "goby/acomms/protobuf/driver_base.proto";

package goby.acomms.protobuf;

message MoshRelayConfig
{
  optional AppBaseConfig base = 1;

  optional string ip_address = 2 [default="127.0.0.1"];
  optional uint32 udp_port = 3;
  optional bool bind = 4;

}

